cylc/uiserver/graphql/tornado_ws.py

```diff
@@ -202,7 +202,8 @@ async def _process_message(self, connection_context, parsed_message):
             return self.on_connection_terminate(connection_context, op_id)
 
         elif op_type == GQL_START:
-            assert isinstance(payload, dict), "The payload must be a dict"
```

Comment:
Python will skip `assert` statements in some situations, this change should be functionally equivalent, but more secure.